_posts/2016-07-09-tomesh-mini.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Will it change significantly? This is (almost) verbatim what is already up on the makerfest website. 
Any adjustment will also have to be updated there.